src/server/db_slice.h

```diff
@@ -405,17 +405,15 @@ class DbSlice {
 
   using ChangeCallback = std::function<void(DbIndex, const ChangeReq&)>;
 
-  //! Registers the callback to be called for each change.
-  //! Returns the registration id which is also the unique version of the dbslice
-  //! at a time of the call.
+  // Called before every access to an entry with a FindMutable call. Returns version
```

Comment:
could you add a comment that this function should be called in the global transaction only